archetypes/helidon/src/main/archetype/mp/database/files/src/test/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,16 @@
+# Microprofile server properties
```

Comment:
Shouldn't this file be removed since you have a template for it?

Reply:
This file contains a config for tests with values that will not change, so I think this file should not be deleted.